search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -245,13 +251,19 @@
   (let [{:keys [id summary short-name project-sn update-time insert-time provider-id
                 science-keywords-flat entry-title opendata-format start-date end-date
                 related-urls publication-references personnel shapes archive-center
-                granule-start-date-stored granule-end-date-stored]} item
+                revision-date2 granule-start-date-stored granule-end-date-stored]} item
         issued-time (get-issued-modified-time insert-time granule-start-date-stored)
+        ;; if issued-time is default date, set it to nil.
+        issued-time (when-not (= issued-time (str umm-spec-date-util/parsed-default-date))
```

Comment:
Rather than two bindings, I think something like this would be easier to understand

```
(if (= issued-time (str umm-spec-date-util/parsed-default-date)
  (get-issued-modified-time insert-time granule-start-date-stored)
  nil)
```

Reply:
That won't work. issued-time doesn't contain anything before calling get-issued-modified-time.

Reply:
revision-date2 is what's being stored. have to use it.

Reply:
Ah yeah my bad. In that case it probably deserves its own function